insights/parsers/ld_library_path.py

```diff
@@ -64,3 +64,31 @@ def parse_content(self, content):
             raise SkipComponent("LD_LIBRARY_PATH not set.")
 
         self.extend(llds)
+
+
+@parser(Specs.ld_library_path_of_global)
+class GlobalLdLibraryPath(Parser):
```

Comment:
The class name `GlobalLdLibraryPath` looks more like that it will provide the value of the `LD_LIBRARY_PATH`, however, it just returns the files that `exported` this variable, but not its actual value.

Would you mind renaming it more relevant?

Reply:
Thanks! Updated.